config/config_test.go

```diff
@@ -107,6 +107,24 @@ func TestNewConfig(t *testing.T) {
 		assert.Equal(t, testConfig, actual)
 	})
 
+	t.Run("success with empty configs", func(t *testing.T) {
+		cnfPath := filepath.Join(testDir, "emptyConfigs")
```

Comment:
If `current` cannot be deleted, empty config pattern will not be happen.
So I think we don't need this test case.

Reply:
I deleted this test